pages/collections/create.tsx

```diff
@@ -436,6 +436,15 @@ const CollectionCreationPage: NextPage = () => {
               >
                 {transactionHash}
               </Anchor>
+              <Button className="mt-2">
+                <Anchor
+                  className="text-white"
+                  external
+                  href={`https://testnet.publicawesome.dev/launchpad/${minterContractAddress as string}`}
+                >
```

Comment:
Can we get this link from an environment variable? So we can use the same thing in mainnet website.